### PR TITLE
Set Selenium version to 2.53.1

### PR DESCRIPTION
**Pull request by marjev**
Here's a proposed fix for recent test failures that I've been experiencing on ecommerce PRs. See #965 and #970 for Travis build failure info.

This PR is influenced by this [comment](http://stackoverflow.com/a/38164528/5859065).
Also, I noticed that in edx-platform repo, Selenium is set to `2.53.1`.

@edx/ecommerce Can someone please take a look?

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @marjev! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?number=971&repo=edx%2Fecommerce

**Comment by clintonb** [comment]
I have cleared the pip cache and restarted https://travis-ci.org/edx/ecommerce/builds/168289247. I would prefer to figure out the cause of the issue, and either adjust the Firefox version on Travis or _upgrade_ to a later version of Selenium. My concern with pinning to an older version of Selenium is that we end up stuck there.

@marjev, did you try upgrading to Selenium 3.0.1?

**Comment by clintonb** [comment]
It might also help to instruct Travis to always use the latest version of Firefox: https://docs.travis-ci.com/user/firefox/.

**Comment by marjev** [comment]
@clintonb No, I haven't tried to upgrade to Selenium 3.0.1? I can try that.

I can try using the latest version of Firefox. I will let you know how it goes.

File: .travis.yml [hunk 1]
```diff
@@ -19,11 +19,19 @@ before_install:
     - "export DISPLAY=:99.0"
     - "sh -e /etc/init.d/xvfb start"
     - "export DJANGO_SETTINGS_MODULE=ecommerce.settings.test"
-    - "export PATH=$PATH:/usr/bin"
 install:
     - pip install -U codecov
     - pip install -U pip wheel
     - make requirements
+    - "mkdir /root/bin"
```
Comment (clintonb) [inline .travis.yml:26]: This failed on Travis. This action make require root access. `sudo` is not available for tests run on container-based build environments, which ours are. See https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments for details.

Comment (marjev) [reply .travis.yml:26]: Closing this in favor of your PR.

Thanks for working on this!

**Pull request closed without merging**